stream_alert/rule_processor/handler.py

```diff
@@ -93,6 +97,12 @@ def run(self, event):
         if not records:
             return False
 
+        threat_intel_config = dict()
+        if (self.__config.get('threat_intel')
```

Comment:
Do you need those parens?

Reply:
Yes, I do need those parenthesizes. Pylint is very helpful and it will warm if there is unnecessary parenthesizes.